sormas-backend/src/main/java/de/symeda/sormas/backend/labmessage/LabMessageFacadeEjb.java

```diff
@@ -245,12 +245,13 @@ public void bulkAssignLabMessages(List<String> uuids, UserReferenceDto userRef)
 	}
 
 	@Override
+    @RolesAllowed({
```

Comment:
I think this method should be restricted to users with the `LAB_MESSAGES` right. Probably adding these roles here and opening it more it's not the right solution

Reply:
The bug is not reproducible anymore. The already fixed bugs solved the Sample creation bug too.